solidity/contracts/helpers/TestLiquidityPoolV1Converter.sol

```diff
@@ -16,4 +18,12 @@ contract TestLiquidityPoolConverter is LiquidityPoolV1Converter {
     function setEtherToken(IEtherToken _etherToken) public {
         etherToken = _etherToken;
     }
+
+    function time() internal view override returns (uint256) {
+        return currentTime != 0 ? currentTime : now;
+    }
+
+    function setTime(uint256 _currentTime) public {
+        currentTime = _currentTime;
+    }
 }
```

Comment:
Due to `uint256 timeElapsed = time() - prevAverageRateUpdateTime` in the real contract, we would probably be better off here with:
```
    function increaseTime(uint256 _seconds) public {
        currentTime += _seconds;
    }
```
Because setting the current time to a value lower than `prevAverageRateUpdateTime` would cause the statement above to underflow.

Reply:
unless the tests also update `prevAverageRateUpdateTime` accordingly - better keep this flexible.